http_urls.py

```diff
@@ -1,13 +1,25 @@
 from django.urls import include, path
+
 from rest_framework import routers
 
 from developer.views.application import ApplicationViewSet
+from developer.views.application import ApplicationHiddenDetailView
 
 app_name = 'developer'
 
 router = routers.SimpleRouter()
 router.register(r'application', ApplicationViewSet, basename='application')
 
 urlpatterns = [
-    path('', include(router.urls)),
+    path
+    (
+        'application/secret_data/',
+        ApplicationHiddenDetailView.as_view(),
+        name='hidden-detail',
+    ),
+    path
```

Comment:
Why did you break this? Any specific reason? This wasn't exceeding 79 chars.

Reply:
In most of the function calls still there is space around the `=` sign. That should be removed. We should integrate a lint and another convention checker to ensure it.